about/glossary.md

```diff
@@ -0,0 +1,45 @@
+# Glossary
+
+## Plone terms
+
+:::{glossary}
+
+Diazo
+  [Diazo theme engine guide](http://docs.diazo.org/en/latest/)
+  Diazo allows you to apply a theme contained in static HTML to a dynamic website created using any server-side technology.
+
+TTW
+  Through-The-Web. Changes made in browser. Be aware to create a Plone add-on for permanence. See doc.plone.org for add-on.
+
+Volto
+  ReactJS frontend of Plone
+
+ZODB
+  [A native object database for Python](http://www.zodb.org/en/latest/).
+
+ZCML
+  The [Zope Configuration Mark-up Language](https://docs.plone.org/develop/addons/components/zcml.html).
+
+:::
+
+
+
+## Non-Plone terms
+
+:::{glossary}
+
+Ansible
+  [Ansible](https://www.ansible.com/) is an open source automation platform.
+  Ansible can help you with configuration management, application deployment, task automation.
+
+Elasticsearch
+  [Elasticsearch](https://www.elastic.co/elastic-stack/) is an open source enterprise search platform.
+
+Solr
+  [Solr](https://solr.apache.org/) is an open source enterprise search platform.
+
+
+ReactJS
+  A JavaScript library for building user interfaces
+
+:::
```

Comment:
This file is superseded by PR #512. After it is merged, and after @stevepiercy moves the global glossary to `docs/glossary.md`, compare to see if anything was missed.